material-stepper/src/main/java/com/stepstone/stepper/type/TabsStepperType.java

```diff
@@ -57,11 +57,11 @@ public void onStepSelected(int newStepPosition) {
      * {@inheritDoc}
      */
     @Override
-    public void onNewAdapter(@NonNull AbstractStepAdapter<?> stepAdapter) {
+    public void onNewAdapter(@NonNull StepAdapter<?> stepAdapter) {
         List<Integer> titles = new ArrayList<>();
         final int stepCount = stepAdapter.getCount();
         for (int i = 0; i < stepCount; i++) {
-            final Step step = stepAdapter.getItem(i);
+            final Step step = (Step) stepAdapter.createStep(i);
```

Comment:
The object returned from _stepAdapter.createStep(i)_ is casted to _Step_. This seems kind of weird since I would assume the _createStep_ method to actually return a _Step_ instance. I'm not sure if I explained this well enough :P